login-fire-social.html

```diff
@@ -66,24 +67,26 @@
       }
     </style>
 
-    <template is="dom-if" if="[[onlySignOut]]">
+
+
+    <template is="dom-if" if="[[_showOneButton]]">
```

Comment:
I think duplicate import is fine. HTML import loads only one time the target file. With lazy loading I guess Service Worker will serve the file from cache and at the compile step (if your app has one) some plugins rollup the dependencies to ensure to add the source code only once.

I'm not an expert on this topic. I just can say I prefer my user reach the app slowly than not at all.